cmd/kubeadm/app/util/user.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os/user"
+
+const (
+	sudoUser = "0"
+)
+
+func IsRoot() (*user.User, bool) {
+	u, err := user.Current()
```

Comment:
No please don't do this.
It's a total PR blocker for me, since it will bring in a lot of C code which will break the static link and some OSes like GCI.

For more information, see: #26028. We should probably document somewhere that use of `os/user` is a big no-no if not _really_ required.

Use `os.Getuid()` instead.